source/guide_syncthing.rst

```diff
@@ -41,22 +41,8 @@ verify the checksum specified in the respective ``.sha256`` file and finally ext
 ::
 
   [isabell@stardust ~]$ mkdir ~/syncthing
-  [isabell@stardust ~]$ wget -O syncthing/syncthing.tar.gz https://github.com/syncthing/syncthing/releases/download/v1.1.3/syncthing-linux-amd64-v1.1.3.tar.gz
-  --2019-05-30 19:16:10--  https://github.com/syncthing/syncthing/releases/download/v1.1.3/syncthing-linux-amd64-v1.1.3.tar.gz
-  Resolving github.com (github.com)... 140.82.118.3
-  Connecting to github.com (github.com)|140.82.118.3|:443... connected.
-  HTTP request sent, awaiting response... 302 Found
-  Location: https://github-production-release-asset-2e65be.s3.amazonaws.com/14712850/0ea39200-71b9-11e9-8b63-1c457246edfa?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20190530%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20190530T171610Z&X-Amz-Expires=300&X-Amz-Signature=d579bd937f01b31321b38a436e2ea22ef5838fa4f821e4e6a66648f3d4631710&X-Amz-SignedHeaders=host&actor_id=0&response-content-disposition=attachment%3B%20filename%3Dsyncthing-linux-amd64-v1.1.3.tar.gz&response-content-type=application%2Foctet-stream [following]
-  --2019-05-30 19:16:10--  https://github-production-release-asset-2e65be.s3.amazonaws.com/14712850/0ea39200-71b9-11e9-8b63-1c457246edfa?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20190530%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20190530T171610Z&X-Amz-Expires=300&X-Amz-Signature=d579bd937f01b31321b38a436e2ea22ef5838fa4f821e4e6a66648f3d4631710&X-Amz-SignedHeaders=host&actor_id=0&response-content-disposition=attachment%3B%20filename%3Dsyncthing-linux-amd64-v1.1.3.tar.gz&response-content-type=application%2Foctet-stream
-  Resolving github-production-release-asset-2e65be.s3.amazonaws.com (github-production-release-asset-2e65be.s3.amazonaws.com)... 52.216.162.59
-  Connecting to github-production-release-asset-2e65be.s3.amazonaws.com (github-production-release-asset-2e65be.s3.amazonaws.com)|52.216.162.59|:443... connected.
-  HTTP request sent, awaiting response... 200 OK
-  Length: 8690658 (8.3M) [application/octet-stream]
-  Saving to: ‘syncthing/syncthing.tar.gz’
-
-  100%[=======================================================================================================================================>] 8,690,658   8.17MB/s   in 1.0s
-
-  2019-05-30 19:16:12 (8.17 MB/s) - ‘syncthing/syncthing.tar.gz’ saved [8690658/8690658]
+  [isabell@stardust ~]$ wget -O syncthing/syncthing.tar.gz https://github.com/syncthing/syncthing/releases/download/v1.1.4/syncthing-linux-amd64-v1.1.4.tar.gz
+[...]
```

Comment:
This line isn't indented correctly, [breaking the code block](https://deploy-preview-431--uberlab.netlify.com/guide_syncthing.html#installation). Can you leave a couple of wget output in the snippet, so users know what to expect, please? :) Maybe the last few, because they are version-agnostic.

Reply:
You're right, changed it.